src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -401,6 +405,81 @@ protected override void Dispose(bool disposing)
         base.Dispose(disposing);
     }
 
+    private static Sampler GetSampler(IConfiguration configuration, Sampler? stateSampler)
+    {
+        Sampler? sampler = null;
+
+        if (stateSampler != null)
+        {
+            sampler = stateSampler;
+        }
+
+        if (configuration.TryGetStringValue(TracesSamplerConfigKey, out var configValue))
+        {
+            if (sampler != null)
+            {
+                OpenTelemetrySdkEventSource.Log.TracerProviderSdkEvent(
+                    $"Trace sampler configuration value '{configValue}' has been ignored because a value '{sampler.GetType().FullName}' was set previously.");
```

Comment:
```suggestion
                    $"Trace sampler configuration value '{configValue}' has been ignored because a value '{sampler.GetType().FullName}' was set programmatically.");
```